apps/predictions/lib/predictions_pub_sub.ex

```diff
@@ -0,0 +1,239 @@
+defmodule Predictions.PredictionsPubSub do
+  @moduledoc """
+  Allow channels to subscribe to prediction streams, which are collected into a
+  Map with keys representing route-stop-direction triads as
+  "<route-id>-<parent-stop-id>-<direction-id>"
```

Comment:
**super-duper nitpick (non-blocking):** this now matches the format from `prediction_key/3`, but the actual implementation uses colons rather than hyphens.

Also just as a thought, maybe you could just use tuples instead of strings as the map keys? Unless there's some place they need to be serialized as strings, but maybe even them you could do that as a separate step.